ttnn/cpp/ttnn/operations/moreh/moreh_linear_backward/device/moreh_linear_backward_multi_core_program_factory.cpp

```diff
@@ -70,34 +58,36 @@ operation::ProgramWithCallbacks moreh_bias_backward_multi_core_h(const Tensor &o
     const uint32_t out0_t = 1;
     const uint32_t im0_t = 1;
     const uint32_t im1_t = 1;
+    auto cb_data_format = tt::tt_metal::datatype_to_dataformat_converter(output_grad.get_dtype());
```

Comment:
```suggestion
    using namespace tt;
    using namespace tt::tt_metal;
    auto cb_data_format = datatype_to_dataformat_converter(output_grad.get_dtype());
```
The TT examples are using `using namespace`. I think it would be good for us to adopt that as well.

Reply:
resolved in [9b40fca](https://github.com/tenstorrent/tt-metal/pull/12812/commits/9b40fca6fe2a092b7588eaeac2d24d998ecb517d)